controllers/apartments_controller.rb

```diff
@@ -0,0 +1,46 @@
+get '/apartments' do
+  @apartments = Apartment.all
+  erb :"apartments/index"
+end
+
+get '/apartments/new' do
+  erb :"apartments/new"
+end
+
+post '/apartments/new' do
+  @address = params[:address]
+  @rent = params[:monthly_rent]
+  @sqft = params[:sqft]
+  @beds = params[:num_beds]
+  @baths = params[:num_baths]
```

Comment:
No need to use instance vars here, a local var `baths` (etc) would do just fine. Instance vars are only needed if you're rendering a view, and need to pass that data to the view. A redirect doesn't render, it triggers a new request to a different route.